### ISSUE TITLE
[BUG] YouTube features missing since 3.0 update.

### ISSUE THREAD

**Issue by wreaver** [SevenTV/Extension#671]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### This issue exists in the latest nightly version

- [X] I am using the latest nightly version

### What browsers are you seeing the problem on?

Chrome

### Current Behavior

Many YouTube features have been missing since the 3.0 update. Which are the following:
*True theater mode
*Emote menu
*Emote hover preview
Only emotes seem to be working, every other feature has been broken since then.

### Expected Behavior

*Being able to switch to true theater mode.
*Being able to open the emote menu.
*Being able to hover over emotes to see their names.

### Steps To Reproduce

Just enter a YouTube livestream and look at the chat. Emotes work, but nothing else does.

**Comment by spikymoth** [comment]
I really miss the True Theater mode on YouTube - when it worked, it was a killer feature that made the extension worth using all on its own. It was very flaky on my laptop for some reason - it would show up maybe 50% of the time if that. But I would love to see it return in any form, because YouTube's stream layout is garbage without it.

**Comment by wreaver** [comment]
> I really miss the True Theater mode on YouTube - when it worked, it was a killer feature that made the extension worth using all on its own. It was very flaky on my laptop for some reason - it would show up maybe 50% of the time if that. But I would love to see it return in any form, because YouTube's stream layout is garbage without it.

Yeah, used to be the same for me too. The extension would only work on YT around 70% of the time. I think it's more consistent now, but I rather take the inconsistency with more features from before than the lack of features now.

**Comment by hugeblank** [comment]
Even emotes barely work. Not sure if you want to add #389 to the list, but if you do I'd be willing to close it.

**Comment by MiguelMachado-dev** [comment]
Sad to see youtube abandoned on this.
Extension does not work there. 0%
